scylla-cql/src/frame/response/cql_to_rust.rs

```diff
@@ -941,11 +941,6 @@ mod tests {
         );
     }
 
-    // Enabling `expect_used` clippy lint,
-    // validates that `derive(FromRow)` macro definition does do not violates such rule under the hood.
-    // Could be removed after such rule will be applied for the whole crate.
-    // <https://rust-lang.github.io/rust-clippy/master/index.html#/expect_used>
-    #[deny(clippy::expect_used)]
```

Comment:
Following our discussion, I agreed with the value of such possible linting errors which could appear under the macro definition.
So as we dont want to prevent using `expect` or `unwrap` under the macro definition, this line of code just does not make sense.